providers/shared/metaparameters/links/id.ftl

```diff
@@ -0,0 +1,12 @@
+[#ftl]
+
+[@addMetaparameter
+    type=LINK_METAPARAMETER_TYPE
+    pluralType="Links"
+    properties=[
+        {
+                "Type"  : "Description",
+                "Value" : "A relationship between Components."
+        }]
+    attributes=linkChildrenConfiguration
+/]
```

Comment:
This is probably more where linkChildrenConfiguration should exist, as long as we load these early on then it shouldn't impact other usage of linkChildrenConfugrration